src/debug/test-utils.js

```diff
@@ -76,8 +76,14 @@ export async function createDebugEngine(DATA_ROOT, LANGUAGE_SERVER_ROOT, LANGUAG
     if (!fs.isDirectorySync(LANGUAGE_SERVER_ROOT)) {
         throw new Error(`${LANGUAGE_SERVER_ROOT} doesn't exist.`);
     }
-
-    const promise1 = startDebugServer(DATA_ROOT, config.logLevel || 'FINE');
+    var defaultSettings={
```

Comment:
Turn off the logLevel from FINE to INFO for some tests which has very large vscode sesssions. See my previous code https://github.com/vscjavaci/java-debug-ci/blob/master/src/debug/stacktrace-test.js


  get logLevel() {
        return "INFO";
    }